File: SoftInvPersistance/src/main/java/pe/edu/pucp/softinv/dao/DisponibilidadDAO.java
package pe.edu.pucp.softinv.dao;
import pe.edu.pucp.softinv.model.Disponibilidad.DisponibilidadDTO;
import pe.edu.pucp.softinv.model.Personas.ClienteDTO;



public interface DisponibilidadDAO {

    public Integer insertar(DisponibilidadDTO disponibilidad);
    public Integer modificar(DisponibilidadDTO disponibilidad);
    public Integer eliminar(DisponibilidadDTO disponibilidad);
}

File: C#/SoftInvPersistance/DAO/DisponibilidadDAO.cs
using softinv.model;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SoftInv.DAO
{
    public interface DisponibilidadDAO
    {
        int insertar(DisponibilidadDTO disponibilidad);

        int modificar(DisponibilidadDTO disponibilidad);

        int eliminar(DisponibilidadDTO disponibilidad);

        DisponibilidadDTO obtenerPorId(int disponibilidadId);

    }
}
